Makefile

```diff
@@ -951,7 +951,111 @@ ifeq "$(OPENMP)" "true"
 endif
 
 
-pio_test:
+openacc_test : 
+ifeq "$(OPENACC)" "true"
+	@#
+	@# First ensure that both FFLAGS_ACC and CFLAGS_ACC are not blank
+	@# If these are not set for a target, then OpenACC most likely cannot compile
+	@#
+	@echo "Checking if FFLAGS_ACC and CFLAGS_ACC are defined for [$(BUILD_TARGET)]..."
+	@if ([ -z "$(FFLAGS_ACC)" ] && [ -z "$(CFLAGS_ACC)" ]); then \
+	   (echo "*********************************************************"; \
+	    echo "ERROR: OPENACC=true was specified, but [$(BUILD_TARGET)] build target does not seem to support OpenACC:"; \
+	    echo "    FFLAGS_ACC and CFLAGS_ACC are both undefined for [$(BUILD_TARGET)] in the top-level Makefile."; \
+	    echo "Please set these variables to appropriate OpenACC compilation flags in the [$(BUILD_TARGET)] target to enable OpenACC support."; \
+	    echo "*********************************************************"; exit 1)\
+	else \
+	   echo "=> FFLAGS_ACC or CFLAGS_ACC defined"; \
+	fi
+
+	@#
+	@# Create test C and FORTRAN programs that look for OpenACC header file and parallelize a loop
```

Comment:
It's a minor stylistic issue, but these days I think Fortran is most often written with just a capital `F` (see, e.g., the Fortran 2003 draft standard), while all-caps seems to have been the prevalant style in decades past.

Reply:
Thanks for the correction!